source/_docs/redis.md

```diff
@@ -36,7 +36,11 @@ For detailed information, see [Installing Redis on WordPress](/docs/wordpress-re
 
 1. Enable the Redis cache server from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**.
 
-2. Add the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
+2. Install and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
+
+    <div class="alert alert-info">
+    <h4 class="info">Note</h4><p markdown="1">You **Must** activate the plugin before proceeding.

```

Comment:
Drupal people call these modules :) There will be confusion if you call them plugins, as Drupal 8 also has plugins that do differnt things.